### PR TITLE
Adding a "beta" flag and moving to cloud.build.io

### PR DESCRIPTION
**Pull request by jprobinson**
Also bumping up to newer versions of Go and dropping the vendor dir.

🤞 this works

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      event: tag
+
+  publish_latest:
+    <<: *publish_config
+    auto_tag: true
+
+  slack:
+    image: plugins/slack
+    channel: dv-notifications
+    webhook: "$$SLACK_WEBHOOK"
```
Comment (tonglil) [inline .drone.yml]: This isn't necessary AFAIK, the secret should set the variable already.

File: .travis.yml [hunk 2]
```diff
@@ -1,20 +0,0 @@
-language: go
```
Comment (oliver-nyt) [inline .travis.yml]: 👍

File: .drone.yml (hunk 1 above)
Comment (oliver-nyt) [reply .drone.yml]: And I'm not sure this even works (as in, will it evaluate the env variable to populate that property).

I think Tony is right, you should be fine as long as the env variable with the secret is `SLACK_WEBHOOK` (which it is).

Comment (jprobinson) [reply .drone.yml]: I've swapped it out to match the docs here: http://plugins.drone.io/drone-plugins/drone-slack/

these also contradict the other secrets docs so this should be fun

**Comment by tonglil** [comment]
For some reason it's running `test` step last. Might due to syntax change.

**Comment by jprobinson** [comment]
oof you're right? WTF? 'test' is the first step in the pipeline. 

Any suggestions?

**Comment by oliver-nyt** [comment]
Oh, maybe that's cause test is the only step you run that's not using the funky `&` to reference the config.

**Comment by jprobinson** [comment]
the yaml variables seemed to have messed up the ordering. all fixed now and 🤞 the secrets play nice on merge. If not, brace for another PR 😓

**Pull request merged**